### DISCUSSION TITLE
Option to specify extra folders to be migrated

### DISCUSSION THREAD

**Discussion by NotActuallyTerry** [belowaverage-org/SuperGrate#82] [Ideas]
**Is your feature request related to a problem? Please describe.**
My current project involves migrating users onto newer PCs. We have a couple of files that are created for each user within their AppData folders.

**Describe the solution you'd like**
An option to specify additional folders to include in the profile backup.

**Describe alternatives you've considered**
I'm currently looking into writing a script to copy these to a directory in Documents. However, having these in the expected location on first login would help significantly.

**Comment by krisdb2009** [comment]
Have you tried placing the files before running a backup / restore?

**Comment by NotActuallyTerry** [comment]
Well yes that would work, but it'd be nice to avoid having to use helper scripts to do that work
